OMCompiler/Compiler/Template/CodegenFMUCommon.tpl

```diff
@@ -58,11 +58,10 @@ template ModelExchange(SimCode simCode, list<String> sourceFiles)
 match simCode
 case SIMCODE(__) then
   let modelIdentifier = modelNamePrefix(simCode)
-  let pdd = providesDirectionalDerivative(simCode)
+  let pdd = if providesDirectionalDerivative(simCode) then ' providesDirectionalDerivative=true' else ''
```

Comment:
In XML terms it should be: ' providesDirectionalDerivative = "true"', but also to keep the same in the tests.